File: spec/code_samples/Python python3/orders/post.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

payload = "{\"exchange\":\"kraken\",\"keyId\":\"string\",\"price\":0,\"volume\":0,\"side\":\"buy\",\"pair\":\"btc_usd\"}"

headers = { 'content-type': "application/json" }

conn.request("POST", "/orders", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node unirest/orders/post.js
var unirest = require("unirest");

var req = unirest("POST", "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/orders");

req.headers({
  "content-type": "application/json"
});

req.type("json");
req.send({
  "exchange": "kraken",
  "keyId": "string",
  "price": 0,
  "volume": 0,
  "side": "buy",
  "pair": "btc_usd"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
